.travis.yml

```diff
@@ -4,6 +4,7 @@ language: node_js
 env:
   global:
     - TMPDIR=/tmp
+    - COVERALLS_REPO_TOKEN=vKN3jjhAOwxkv9HG0VBX4EYIlWLPwiJ9d
```

Comment:
Is there any additional work needed to get it to show up at [coveralls site](https://coveralls.io/github/mozilla/fxa-auth-server)?